pyshelf/configure.py

```diff
@@ -14,8 +14,12 @@ def app(app):
             "accessKey": val.get("accessKey"),
```

Comment:
This doesn't appear to be fixed.  I mean, it is nested under buckets, but the logic problem still exists.  With the way it works right now it will pass validation if any bucket before the last one doesn't have accessKey or secretKey.

Can you write a test for this as well?